resources/css/style.css

```diff
@@ -45,9 +45,22 @@
 .title-get-involved {
   font-family: -apple-system, BlinkMacSystemFont, Roboto, sans-serif;
   font-size: 5em;
-  font-weight: 100;
+  font-weight: 300;
   text-align: center;
 }
+.row .description {
+  font-family: Roboto;
+  font-size: 1.6em;
+  text-align: justify;
+  font-weight: 300;    
```

Comment:
you can also run "coala" and it fixes everything for you automatically, if you have it installed :)